packages/react-instantsearch-hooks-dom/src/ui/CurrentRefinements.tsx

```diff
@@ -15,26 +15,17 @@ export type CurrentRefinementsProps = React.HTMLAttributes<HTMLDivElement> & {
       Record<string, unknown>
   >;
   onRemove?(refinement: CurrentRefinementsConnectorParamsRefinement): void;
-  hasRefinements?: boolean;
 };
 
 export type CurrentRefinementsClassNames = {
   /**
    * Class names to apply to the root element
    */
   root: string;
-  /**
-   * Class names to apply to the root element when there are no refinements possible
-   */
-  rootNoRefinement: string;
```

Comment:
it feels weird to remove information that could be useful, and that exists in the existing React InstantSearch DOM version. I'd rather add it to the specs and InstantSearch.js

Reply:
as you want, for me leaving the implementation makes more sense, rather than adding it back later

Reply:
You're right, I reverted the change and I'll work on adding this to the spec right after this is merged.